apps/dashboard-service/src/dashboard/dto/tree-restoration-goal.dto.ts

```diff
@@ -0,0 +1,74 @@
+import { JsonApiDto } from "@terramatch-microservices/common/decorators";
+import { ApiProperty } from "@nestjs/swagger";
+import { populateDto } from "@terramatch-microservices/common/dto/json-api-attributes";
+
+class TreeRestorationData {
+  @ApiProperty({
+    description: "Due date for this restoration data",
+    type: String,
+    example: "2024-01-01T00:00:00.000000Z"
+  })
+  dueDate: string;
```

Comment:
This should be typed as a `Date`. See the createdAt / updatedAt examples in entity DTOs.